assets/src/web-stories-block/block/block-types/selected-stories/storyPicker.js

```diff
@@ -0,0 +1,341 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Button, Modal } from '@wordpress/components';
+import { useState, useEffect, useMemo, useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import useApi from '../../api/useApi';
+import { VIEW_STYLE, STORY_STATUSES } from '../../constants';
+import { useStoryView } from '../../../../dashboard/utils';
+import { ScrollToTop, Layout } from '../../../../dashboard/components';
+import { useConfig } from '../../config';
+import SelectStories from './selectStories';
+import SortStories from './sortStories';
+import LoaderContainer from './components/loaderContainer';
+
+const StoryPickerModal = styled(Modal)`
+  width: calc(100% - 16px - 16px);
+  height: calc(100% - 60px - 60px);
+  position: relative;
+  overflow: hidden;
+`;
+
+const ModalContent = styled.div`
+  position: relative;
+  height: calc(100% - 93px);
+  margin: -24px -24px 0;
+  padding: 0;
+  overflow-y: scroll;
+`;
+
+const ModalContentInner = styled.div`
+  padding: 12px 24px 24px;
+`;
+
+const ModalFooter = styled.div`
+  position: fixed;
+  bottom: 0;
+  z-index: 2;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #f3f3f3;
```

Comment:
There are lots of hardcoded colors here. Are should these values be in a theme somewhere.